protoc_plugin/lib/src/custom_service_generator.dart

```diff
@@ -160,12 +160,12 @@ class _CustomApiMethod {
     final apiName = convertApiName(_dartName);
     final serviceName = convertServiceName(_serviceName);
     out.addBlock(
-        '$_clientReturnType $apiName($_argumentType request, {$coreImportPrefix.bool toastMessage = true, $coreImportPrefix.bool throwError = true}) async {',
+        '$_clientReturnType $apiName($_argumentType request, {$coreImportPrefix.bool toastMessage = true, $coreImportPrefix.bool throwError = true, $coreImportPrefix.Map<$coreImportPrefix.String, $coreImportPrefix.String>? headers}) async {',
```

Comment:
In what scenarios would it be necessary to pass headers?

Reply:
sorry for late reply, got this feature, where we need to help user with china phone number only to bind their account with email, so we need to parse the token through the header when the user is not logined

https://github.com/olachat/partying-app/pull/2025